OracleSOASuite/dockerfiles/12.2.1.4/README.md

```diff
@@ -1,82 +1,186 @@
 SOA on Docker
 =============
 
-Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle Fusion Middleware Infrastructure 12.2.1.4.0.
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server), and Oracle Fusion Middleware Infrastructure 12.2.1.4.0.
 
-At the end of this configuration there may be 3 containers running: 
-1. Oracle Database Container (Optional: only when RCU schema is created in a database running in a container) 
-2. Oracle Weblogic Administration Server Container 
-3. Oracle Weblogic Managed Server Container (Oracle SOA Server or Oracle Service Bus Server)
+At the end of this configuration there may be 4 containers running:
```

Comment:
Perhaps try,"...there will be at least three running containers:"

Reply:
Updated it as "there will be at least two running containers", because for a "soa" domain type if db is is not in container, then there will be only two containers (one for admin server and one for soa managed server).